jax/_src/numpy/fft.py

```diff
@@ -444,15 +444,162 @@ def _fft_core_2d(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return _fft_core(func_name, fft_type, a, s, axes, norm)
 
 
-@implements(np.fft.fft2)
 def fft2(a: ArrayLike, s: Shape | None = None, axes: Sequence[int] = (-2,-1),
          norm: str | None = None) -> Array:
+  r"""Compute a two-dimensional discrete Fourier transform along given axes.
+
+  JAX implementation of :func:`numpy.fft.fft2`.
+
+  Args:
+    a: input array.
```

Comment:
Mention that it should have at least two dimensions

Reply:
Done